app/routes.py

```diff
@@ -1,2 +1,346 @@
-from flask import Blueprint
+from app import db
+from app.models import video
+from app.models import customer
+from app.models.customer import Customer
+from app.models.rentals import Rental
+from app.models.video import Video
+from flask import json, request, Blueprint, make_response, jsonify
+from datetime import datetime, timedelta
+from flask_sqlalchemy import SQLAlchemy
+from sqlalchemy import desc, asc
+import os
+from dotenv import load_dotenv
+import requests
+
+load_dotenv()
+
+customer_bp = Blueprint("customers", __name__, url_prefix="/customers")
+video_bp = Blueprint("videos", __name__, url_prefix="/videos")
+rental_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+# CRUD CUSTOMERS
+@customer_bp.route("", methods=["GET"])
+def list_customers():
+    """Retrieves all customers and their related data from database"""
+    customers = Customer.query.all()
+
+    list_of_customers = []
+    if not customers:
+        return jsonify(list_of_customers)
+    for customer in customers: 
+        list_of_customers.append(customer.to_json()) 
+    return jsonify(list_of_customers)
+
+@customer_bp.route("/<customer_id>", methods=["GET"])
+def list_single_customer(customer_id):
+    """Retrieves data of specific customer"""
+    single_customer = Customer.query.get(customer_id)
+    if not single_customer:
+        return make_response({"details": f"There is no customer in the database with ID #{customer_id}"}, 404)
```

Comment:
You can simplify this with `get_or_404`.

```suggestion
    single_customer = Customer.query.get_or_404(customer_id)
```